src/ray/gcs/gcs_server/gcs_resource_manager.cc

```diff
@@ -111,16 +103,8 @@ void GcsResourceManager::DeleteResources(const NodeID &node_id,
       (*resource_map.mutable_items())[entry.first].set_resource_capacity(entry.second);
     }
 
-    auto on_done = [this, node_id,
```

Comment:
Should we remove Update/Delete resources in this case?

Reply:
I think I fail to get what you mean here. Do you mean remove Update/Delete resource in GcsResourceManager?